docker-compose/docker-compose-standard.yml

```diff
@@ -469,12 +469,22 @@ services:
       - ./nginx/api/:/etc/nginx/api_conf.d
       - ./nginx/gateway/api_gateway.conf:/etc/nginx/api_gateway.conf
       - ./nginx/api_json_errors.conf:/etc/nginx/api_json_errors.conf
+
+      # Certificate generated by Letsencypt on the Host. Only fullchain.pem and privkey.pem are needed.
+      # Replace YOUR_DOMAINNAME with the DNS value you have acquired the Certificate for on the Host
+      - /etc/letsencrypt/live/YOUR_DOMAINNAME/fullchain.pem:/etc/letsencrypt/live/YOUR_DOMAINNAME/fullchain.pem
+      - /etc/letsencrypt/live/YOUR_DOMAINNAME/privkey.pem:/etc/letsencrypt/live/YOUR_DOMAINNAME/privkey.pem
```

Comment:
It is necessary that the internal folder of the nginx container contains the word `YOUR_DOMAINNAME`? I understand that it can be replaced, but if you can use something generic (like `secure_domain` or `certificates`) so that it is established as a criterion that it will always be the internal path of the container, I think it would be more practical regardless if the host path changes.